R/declare_assignment.R

```diff
@@ -15,7 +15,7 @@
 #' 
 #' Custom assignment handlers should augment the data frame with an appropriate column for the assignment(s).
 #'
-#' @importFrom rlang quos quo lang_modify eval_tidy !!!
+#' @importFrom rlang quos quo call_modify eval_tidy !!!
```

Comment:
Maybe delete this line? Usually I try to keep the importFroms near the functions that actually use them.